### PR TITLE
Issue #55 Longest Path in a Matrix

### PR DESCRIPTION
**Pull request by keivalya**
## Have you read the Contribution Guidelines?
Yes!
(Write your answer here.)

## Description
A program to find the longest length path in a matrix which is taken as input from the user. The difference between two consecutive elements must be the same for all the elements present in the path.
(Important concept for competitive programming).
(Write your answer here.)

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)
# #55 is resolved in Python
Fixes Issue 
<br/>
(Write your answer above.)

### CODE REVIEW

**Comment by sukritishah15** [comment]
@keivalya Please edit the README.md file in Python folder and provide link to your code as well, whenever you create a PR.

File: Python/LongestPath_Matrix.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+				findLongestFromACell(i, j, mat, dp) 
+
+			result = max(result, dp[i][j]); 
+	return result 
+
+# Given Matrix
+mat = [[1, 2, 9], 
+	[5, 3, 8], 
+	[4, 6, 7]] 
+print("Length of the longest path is ", finLongestOverAll(mat))
```
Comment (sukritishah15) [inline Python/LongestPath_Matrix.py:44]: At the bottom of the file, provide sample input, output, time and space complexity in a single multiline comment.

Comment (keivalya) [reply Python/LongestPath_Matrix.py:44]: Yep. Done. Please check.

Comment (sukritishah15) [reply Python/LongestPath_Matrix.py:44]: NO. In a single multiline comment enclose all 4 things - Sample I/P, O/P, TIME AND SPACE complexity.
I can't see time and space complexity.
Also i/p and o/p are not inside a comment.
Everything in 1 single comment.

Also, you'll have to add link to the file LongestPath_Matrix.py in the README.md file inside Python folder, as per the format given there.

@keivalya

**Pull request merged**